cmd/cdi-controller/controller.go

```diff
@@ -58,6 +72,12 @@ var (
 func init() {
 	// flags
 	flag.StringVar(&masterURL, "server", "", "(Optional) URL address of a remote api server.  Do not set for local clusters.")
+	flag.StringVar(&uploadServerKeyFile, "uploadserver-ca-key", defaultUploadServerKeyFile,"(Optional) Client certificate for Upload Server")
+	flag.StringVar(&uploadServerCertFile, "uploadserver-ca-cert", defaultUploadServerCertFile, "(Optional) Private key for the client certificate for Upload Server")
+	flag.StringVar(&uploadClientKeyFile, "uploadserver-client-ca-key", defaultUploadClientKeyFile, "(Optional) Client certificate for Upload Server Client")
+	flag.StringVar(&uploadClientCertFile, "uploadserver-client-ca-cert", defaultUploadClientCertFile, "(Optional) Private key for the client certificate for Upload Server Client")
+	flag.StringVar(&uploadServerCABundleConfigMap, "uploadserver-ca-config-map", defaultUploadServerCABundleConfigMap, "(Optional) Name of Config Map containing CA Bundle for Upload Server")
+	flag.StringVar(&uploadClientCABundleConfigMap, "uploadserver-client-ca-config-map", defaultUploadClientCABundleConfigMap, "(Optional) Name of Config Map containing CA Bundle for Upload Server Client")
```

Comment:
The big change I would like to discuss in this PR is moving from command line args to environment variables.  We don't use command line args much.  Clearly, some are there, but I don't think we ever invoke them.

I'm thinking that for each relevant private key/cert there will be mandatory environment variables with the path to the mounted file.  Variables named `UPLOADSERVER_KEY_FILE`, `UPLOADSERVER_CERT_FILE`, etc.  What do you think?

Reply:
Thanks for taking a look. Yes environment variables seem better option than command line args. Updated to take in custom args using env variables.